src/Samples/dotnet6server/dotnet6Server.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CoreWCF.ConfigurationManager" Version="0.4.0" />
+    <PackageReference Include="CoreWCF.Http" Version="0.4.0" />
```

Comment:
This sample isn't using Configuration or NetTcp so you don't need to reference either of them.